packages/react/src/progress-indicator/ProgressIndicatorItem.tsx

```diff
@@ -90,6 +95,29 @@ const ProgressIndicatorItem = ({
 
 	const listItemLinkTextSelector = '> span:last-of-type > span:first-of-type';
 
+	if (isActive) {
+		const activeStatusWhiteList: Array<ProgressIndicatorItemStatus> = [
+			'blocked',
+			'doing',
+			'started',
+		];
```

Comment:
Is it worth pulling this outside the component since it's static?

Reply:
moved out and I also export it so it can be used by the test.tsx file